### ISSUE TITLE
A11y_VisualStudioCodeClient_Search_Global replace_Keyboard: Keyboard focus is not visible for Match case button.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#191068]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.
### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Aug2023; #Voiceover; #MAC; #WCAG2.4.7; #Focus Visible.
### Environment and OS details:
Visual Studio Code insider for MAC version 1.82.0
Mac Version: 13.5
Screen reader: Voice Over
### Repro Steps:
1.	Open Visual Studio Code Insiders editor. 
2.	Using keyboard navigation View Menu Bar.
3.	Select search option.
4.	Search and replace global content.
5.	Press Option + Tab to navigate to Match case button select the button and Press Option + Shift + Tab to navigate back to the button and Observe that Keyboard Focus is not visible for Match case button.
### Actual Result:
Keyboard focus is not visible for Match case button.
### Expected Result:
Keyboard focus should be visible for Match case button.
### User Impact:
Keyboard users will face difficulty if Keyboard focus is not visible for Match case button.
### Attachment:

https://github.com/microsoft/vscode/assets/98728920/d6b40255-e22a-423d-9ef5-9921b2202fef
